packages/cli/src/utils.ts

```diff
@@ -328,16 +302,12 @@ export async function setSubscriptionId(
 export function isWorkspaceSupported(workspace: string): boolean {
```

Comment:
I am not sure if there is an API in core to do this workspace checking. Did you check it when you are trying to clean it?

Reply:
And for the following several utils like 'getTeamsAppId', 'getLocalTeamsAppId', I saw someone add comments like 'only used when multi-env is disabled'. I am not sure if these utils are properly used.

Reply:
1. It seems there is a `isValidProject` in core that is similar to `isWorkspaceSupported`. But this pr is mainly for removing `isMultiEnvEnabled`. We can cleanup that later. 2. Good point, I will cleanup these utils too